packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -276,7 +276,10 @@ export async function importBlock(
     const safeBlockHash = this.forkChoice.getJustifiedBlock().executionPayloadBlockHash ?? ZERO_HASH_HEX;
     const finalizedBlockHash = this.forkChoice.getFinalizedBlock().executionPayloadBlockHash ?? ZERO_HASH_HEX;
     if (headBlockHash !== ZERO_HASH_HEX) {
-      this.executionEngine.notifyForkchoiceUpdate(headBlockHash, safeBlockHash, finalizedBlockHash).catch((e) => {
+      // Call and wait on the fcU even though failure of this may not be critical for us to import
+      // ( and hence catch log error). Otherwise this may result in out of order fcU on the execution
+      // engine especially during sync of multiple blocks.
+      await this.executionEngine.notifyForkchoiceUpdate(headBlockHash, safeBlockHash, finalizedBlockHash).catch((e) => {
```

Comment:
maybe add this to a new job queue to make sure EL receive fcu requests in order? if we leave `await` here we'll add some more time to block processing time - below is goerli metric

<img width="797" alt="Screen Shot 2022-09-26 at 08 53 34" src="https://user-images.githubusercontent.com/10568965/192178728-5e31b938-46bc-4706-bf8f-3f9bd0f3fe80.png">

cc @dapplion @wemeetagain

Reply:
making the updates...